src/ui/shared/requests/useNativeBalance.ts

```diff
@@ -29,10 +31,18 @@ function useNativeAddressPosition({
   );
 
   return useMemo(() => {
+    const nativePositions = value?.positions?.filter(
+      (item) =>
+        item.chain === chain.toString() &&
+        item.type === 'asset' &&
+        !item.protocol
+    );
+    invariant(
+      !nativePositions || nativePositions.length <= 1,
+      'multiple native positions'
+    );
```

Comment:
Do we really want to crash here?

Reply:
Also this condition is quite hard to read :)

Reply:
The idea is to make it easier to debug in case anything similar happens again. So we'll have a clear message indicating that we received multiple native positions from the backend api

Reply:
I simplified the invariant condition ✅